libraries/lib-audio-io/AudioIO.h

```diff
@@ -78,7 +78,7 @@ struct AudioIOEvent {
 };
 
 struct AUDIO_IO_API TransportTracks final {
-   SampleTrackConstArray playbackTracks;
+   std::vector<std::shared_ptr<const WideSampleSequence>> playbackTracks;
```

Comment:
Slow down.

You are only setting yourself up for more conflict with stuff I have to do.